server/migrations/20200924140557_version-replace-underscore.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex) {
+
+    return knex.raw(
+        `update app_version
+        set version = replace(version, '_', '.')
+        where version like '%$_%' ESCAPE '$'
+    `)
+};
+
+exports.down = function(knex) {
+
+    return knex.raw(
+        `update app_version
+        set version = replace(version, '.', '_')
+        where version like '%$.%' ESCAPE '$'
```

Comment:
Technically this isn't correct for the `down` migration - it will convert all versions which were originally `.` to `_` as well, which would be bad.  Can we just omit the `down` migration for this one?